samples/cmake/newton-dynamics/README.md

```diff
@@ -0,0 +1,52 @@
+You must clone newton-dynamics first; also because of a bug somewhere, you must run CMake twice:
+
+```
+git clone https://github.com/MADEAPPS/newton-dynamics.git --branch v3.14c --depth=1
+mkdir build
+cd build
+${NXDK_DIR}/usr/bin/nxdk-cmake ..
+${NXDK_DIR}/usr/bin/nxdk-cmake ..
```

Comment:
Why is it necessary to run cmake twice tho?

I did not check if this also happens when building for linux (with these flags)